src/panel_gwalker/_gwalker.py

```diff
@@ -1,56 +1,23 @@
 import asyncio
```

Comment:
With the new implementation of `export` its no longer possible to get the current chart. I.e. I have no way of creating applications that supplement the chart that is currently shown in `GraphicWalker` or that export and save the current chart only.

Please add an attribute `current` or similar to enable me to export the current chart only. Thx.

Reply:
Further more the `export` function does not work. Please fix.

Reply:
Fixed